InformationScripting/src/parsing/QueryParser.h

```diff
@@ -60,10 +61,10 @@ class INFORMATIONSCRIPTING_API QueryParser
 		Type typeOf(const QString& text);
 		QPair<QStringList, QList<QChar> > split(const QString& text, const QList<QChar>& splitChars);
 
-		Query* parseQuery(const QString& text);
-		QList<Query*> parseList(const QString& text);
-		Query* parseOperator(const QString& text, bool connectInput = false);
-		QList<Query*> parseOperatorPart(const QString& text);
+		Query* parseQuery(Query* parent, const QString& text);
```

Comment:
What do you mean they don't set the field? I thought we'd provide a setter in the base class and call it for all queries, in which case a query never needs to set any field (even if it wants to use the executor). Did you have another design in mind?